roles/dtc/common/templates/ndfc_fabric/ebgp_vxlan_fabric/evpn/ebgp_vxlan_fabric_evpn.j2

```diff
@@ -1,7 +1,13 @@
 {# Auto-generated NDFC eBGP VXLAN EVPN config data structure for fabric {{ vxlan.fabric.name }} #}
   ENABLE_EVPN: true
+  ENABLE_L3VNI_NO_VLAN: true
```

Comment:
We need to use the default value here, and not hard code this.

Reply:
Sent email indicating putting this under `vxlan.global` is fine but need to handle both 

* Allow L3VNI w/o VLAN
* Enable L3VNI w/o VLAN